api/v1/paths/students/{osuID}/academic-status.js

```diff
@@ -0,0 +1,31 @@
+const appRoot = require('app-root-path');
+
+const { errorBuilder, errorHandler } = appRoot.require('errors/errors');
+const { openapi: { paths } } = appRoot.require('utils/load-openapi');
+
+const studentsDAO = require('../../../db/oracledb/students-dao');
+
+const get = async (req, res) => {
+  try {
+    const { osuID } = req.params;
+    const { term } = req.query;
+    const termPattern = /^\d{4}0[0-3]{1}$/;
```

Comment:
Could this pattern be added to `openapi.yaml`? This way, express-openapi should be able to handle the error. There would probably need to be a conditional added to the error middleware to handle a violation of this rule to respond with a better error message than the default one.

Reply:
Sounds good. I've added the pattern to `openapi.yaml` but I didn't make any customized change to the error middleware since the skeleton code hasn't been pushed to the skeleton repo. I'll do that once the code is there.